addons/hashes/fnc_parseJSON.sqf

```diff
@@ -27,246 +27,524 @@ Examples:
     (end)
 
 Author:
-    BaerMitUmlaut
+    initramfs
 ---------------------------------------------------------------------------- */
 SCRIPT(parseJSON);
-params ["_json", ["_objectType", 0]];
+params [["_json", "", [""]], ["_objectType", 0]];
+
+// Enable unicode processing
+forceUnicode 0;
+
+/**
+ * Helper function returning the code used for object creation based on the "_objectType" parameter.
```

Comment:
Comment explains _what_ the code does. That's just duplication and not useful. If you want to keep it as separator, just use `// helper function`.

Reply:
I've removed the small helper functions with the original code so there are no additional function declarations anymore. The original rationale for the comment is merely my codestyle of always having a preamble for any function declaration.

I believe the preambles for `_tokenize`, `_reduce` definitely provide additional information (wrt. output format, preconditions, etc). To a lesser extent `_shift` and `_parse` do as well, and it seems a little inconsistent to me for some functions to get a full descriptor and some to not.